### ISSUE TITLE
IOS VoiceOver screen reader gestures support

### ISSUE THREAD

**Issue by csaba2309** [twofas/2fas-ios#91]
### Feature description

I am total blind, I am using VoiceOver screen reader on my iPhone. I want to delete some of my tokens. I want to organize my tokens into groups.
I unable to do these operations in this app with VoiceOver.
Except these two, it is usable with VoiceOver.
It is the same with versions IOS 16 and IOS 17.

### Motivation

Many blind people can use these features.

### Acknowledgements

- [X] This issue is not a duplicate of an existing feature request.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by adocyn** [comment]
@csaba2309 Hi, I'll add it to the backlog but, as the list is long, I can't promise quick resolution. Anyway after we'll deal with that I'll get back to you with some questions about further improvements.

**Comment by pylapp** [comment]
@adocyn I spotted that issue and I am able to bring some small fixes.
It will submit a pull request soon with commits to cherry-pick or merge as they are (or whateevr you want).

**Comment by adocyn** [comment]
@csaba2309 @pylapp Great! I've merged the change and it will be available with 5.4 soon
